PyPortal/PyPortal_Alarm_Clock/code.py

```diff
@@ -73,7 +85,7 @@
 
 icon_file = None
 icon_sprite = None
-celcius = secrets['celcius']
```

Comment:
`celcius ` was originally a boolean in the settings.py file. 

It seems that circuitpython does not support boolean values in toml / getenv(). So this would need to be changed to either use ints like `1` and `0` or strings like "True" and "False" or similar.

Reply:
Thinking a bit more and looking at the code now I think `1` and `0` is the simplest option. That actually shouldn't require changing the code any because the logic that uses the value here will use the truthiness of the number values https://github.com/justmobilize/Adafruit_Learning_System_Guides/blob/06fcc8e04c0050673abf73389fe8f8a2b525fc8c/PyPortal/PyPortal_Alarm_Clock/code.py#L297-L300

Reply:
Ahh that's right on the bools. I've checked for things like lists and dicts, but forgot this simple type.

Reply:
`celcius` is a misspelling: should be `celsius`.

Reply:
Thank you. We'll update the spelling in the next PR.